app/controllers/openid_connect/logout_controller.rb

```diff
@@ -7,11 +7,32 @@ class LogoutController < ApplicationController
     include OpenidConnectRedirectConcern
 
     before_action :set_devise_failure_redirect_for_concurrent_session_logout,
-                  only: [:logout]
+                  only: [:show, :create]
     before_action :confirm_two_factor_authenticated, only: [:delete]
+    skip_before_action :verify_authenticity_token, only: [:create]
 
-    # Handle logout (with confirmation if initiated by relying partner)
-    def logout
+    # +GET+ Handle logout (with confirmation if initiated by relying partner)
+    # @see {OpenID Connect RP-Initiated Logout 1.0 Specification}[https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout]  # rubocop:disable Layout/LineLength
+    def show
+      @logout_form = build_logout_form
```

Comment:
it looks like `show` and `create` have identical implementations. would it make sense to shove the `@logout_form = build_logout_form` logic into a `before_action` so both views have access to them, and then extract the rest of logic into a shared method? (or is that just adding some complication without a lot of benefit? seems possible.)

Reply:
We're discussing something similar offline, but it may be easier to use the same method for the two routes.

Reply:
Updated to have `#create` redirect to `#show`. It doesn't obviate extracting the `build_logout_form` logic, but may be an overoptimization since `#show` and `#delete` actually do different things and may diverge further in the future.